README.md

```diff
@@ -52,8 +52,14 @@ use {
 
 ```lua
 require("nvim-paredit").setup({
+  -- should plugin use default keybindings? (default = false)
```

Comment:
The `default` is actually true

Reply:
Then I explicitly add it to defaults
```lua
M.defaults = {
  -- here
  use_default_keybinds = true,
  cursor_behaviour = "auto", -- remain, follow, auto
  keys = {}
}

```

Reply:
Sure, then this logic will need to change:

https://github.com/julienvincent/nvim-paredit/pull/5/files#diff-2dfa57ae2b56cb817596ba1cb09feb549dacf154b931ce865a380fb617c590fbR39